tmt/base.py

```diff
@@ -742,6 +742,8 @@ class Test(Core, tmt.export.Exportable['Test']):
     _normalize_contact = tmt.utils.LoadFmfKeysMixin._normalize_string_list
     _normalize_component = tmt.utils.LoadFmfKeysMixin._normalize_string_list
 
+    metadata_filename = 'metadata.yaml'
```

Comment:
1. this looks like a constant, therefore I'd put it on a module level, and
2. this does not seem to be the right location. IIUIC, there's only one place where this filename is used, and that's https://github.com/teemtee/tmt/blob/main/tmt/steps/execute/__init__.py#L224.

My two cents, then, I'd make this a `TEST_METADATA_FILENAME` name in `tmt/steps/execute/__init__.py`, just like `TEST_OUTPUT_FILENAME` defined there.